sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
-## 2.1.0-beta.1 (Unreleased)
+## 2.1.0-beta.2 (unreleased)
```

Comment:
missing changelog info, if you't=re preparing for release.

Reply:
This is for previously approved PR to merge into this branch for  April. But for  April Release, I will have separate release PR in few days.